### PR TITLE
ckEthtool state

### PR DESCRIPTION
**Pull request by msciciel**
### What does this PR do?

New state ethtool and small fix for docstrings in ethtool module
### What issues does this PR fix or reference?
#29660
### Tests written?

No

### CODE REVIEW

File: salt/states/ethtool.py [hunk 1]
```diff
@@ -0,0 +1,329 @@
+# -*- coding: utf-8 -*-
+'''
+Module for running ethtool command
+
+.. versionadded:: 2016.3.0
```
Comment (rallytime) [inline salt/states/ethtool.py]: This version should be `Carbon`

**Comment by rallytime** [comment]
Hi @msciciel - I see that you are contributing a new state file here for the `ethtool` execution module, but after reviewing the file, this new state is not stateful and looks more like an execution module as it currently stands. Salt state modules need to contain certain pieces to integrate with the state system correctly. One of the main items is a return dictionary that contains information about the state such as the state name, a result (return value), a changes dictionary, and any applicable comments. States should help users enforce the "state" of a system, rather than implementing specific CLI commands like execution modules do. 

We have a tutorial about how to write state modules [here](https://docs.saltstack.com/en/latest/ref/states/writing.html) that I think will be helpful for you. We won't be able to merge this in as it stands, but if you can update this so it is stateful, we'd be happy to get this in. Please let us know if you have any questions.

**Comment by msciciel** [comment]
I don't know how it was possible that I commited module as state. I picked wrong file from my internal repo and didn't notice.  Of course I'll fix this request and include corrrect file as soon as i reach computer. Sorry for wasted time.

**Comment by rallytime** [comment]
@msciciel No problem! Sounds good. :)

**Comment by msciciel** [comment]
I'have submitted correct file. Please take a look again.

**Comment by rallytime** [comment]
Ah, there we go. There are some pylint errors that need to be cleaned up. Mind grabbing those? 

https://jenkins.saltstack.com/job/PR/job/salt-pr-lint-n/2007/violations/file/salt/states/ethtool.py/

**Pull request merged**